### DISCUSSION TITLE
How to set global css class for specific component?

### DISCUSSION THREAD

**Discussion by ivanjaros** [quasarframework/quasar#17204] [General - Components / Directives / etc]
I want to set default css class for a component, specifically q-tooltip. I want to do this globally - I do not want to do it per each instance, manually, it should simply be a global default.

Based on the conversations here, I was able to alter props globally, but attributes seem to be problematic.

Is there a way to achieve this?

**Comment by yusufkandemir** [comment]
You can use CSS and target `.q-tooltip` and then apply the CSS rules there.
